lib/rust/ensogl/core/src/display/scene.rs

```diff
@@ -589,6 +591,17 @@ impl Renderer {
 // === Layers ===
 // ==============
 
+type RectLayerPartition = Rc<LayerSymbolPartition<rectangle::Shape>>;
+
+
+fn partition_layer<S: display::shape::primitive::system::Shape>(
+    base_layer: &Layer,
+    name: &str,
+) -> Rc<LayerSymbolPartition<S>> {
+    let partition = base_layer.create_symbol_partition::<S>(name);
+    Rc::new(partition)
+}
+
```

Comment:
I wonder if this function is worth keeping. We could "inline" it, simplifying the code readability (one function name less to remember).